src/main/java/hudson/plugins/copyartifact/DownstreamBuildSelector.java

```diff
@@ -177,7 +177,7 @@ protected boolean containsVariable(String str) {
          * @return
          */
         public FormValidation doCheckUpstreamProjectName(
-                @AncestorInPath AbstractProject<?,?> project,
+                @AncestorInPath Job<?,?> project,

```

Comment:
You should skip the validation if `project` is `null` (that is, when configuring a template).